server/src/main/java/org/elasticsearch/index/codec/vectors/ES813Int8FlatVectorFormat.java

```diff
@@ -37,16 +37,15 @@ public class ES813Int8FlatVectorFormat extends KnnVectorsFormat {
     private final FlatVectorsFormat format;
 
     public ES813Int8FlatVectorFormat() {
-        this(null);
+        this(null, 7, false);
     }
 
     /**
      * Sole constructor
      */
-    public ES813Int8FlatVectorFormat(Float confidenceInterval) {
+    public ES813Int8FlatVectorFormat(Float confidenceInterval, int bits, boolean compress) {
```

Comment:
Are all combinations of allowed `bits` and  `compress` allowed/functional? If not, maybe we could add some validation/extract the `compress` value based on `bits`?